build-docker.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
❗️ We should be using a docker plugin to build the image as part of the maven build so CI can build it. Consider the fabric8 docker plugin: https://dmp.fabric8.io

Reply:
Added it to the POM and removed the build script.